proto/buf.gen.gogo.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 plugins:
   - name: gocosmos
     out: ..
-    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
+    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/gogoproto/types/any
```

Comment:
this is causing proto-breaking-change workflow to fail?

Reply:
no, think its the change from tendermint.types" to "cometbft.types.v1" for some types in tendermint client update messages causing the breakage.

would need to be handled by relayer to send the cometbft types when using v10 rpcs. 

There has been some "breakages" like this before and handled by relayers for targeting different node versions. In a perfect world we would bump the proto package version of tendermint client but its often been a grey area. These types are at least for rpc usage only and don't affect protoAny encoding or the like as they are not stored afaik.

This change for protoAny is just moving from cosmos-sdk/codec/types to gogoproto repo. It's mentioned in sdk's UPGRADING.md. codec/types still maintains alias types pointing to gogoproto repo.